aea/cli/common.py

```diff
@@ -179,54 +184,69 @@ def try_to_load_agent_config(ctx: Context, is_exit_on_except: bool = True) -> No
             sys.exit(1)
 
 
-def _try_to_load_protocols(ctx: Context):
-    for protocol_public_id in ctx.agent_config.protocols:
-        protocol_name = protocol_public_id.name
-        protocol_author = protocol_public_id.author
-        logger.debug("Processing protocol {}".format(protocol_public_id))
-        protocol_dir = Path(
-            "vendor", protocol_public_id.author, "protocols", protocol_name
-        )
-        if not protocol_dir.exists():
-            protocol_dir = Path("protocols", protocol_name)
+def _load_env_file(env_file: str):
+    """
+    Load the content of the environment file into the process environment.
 
+    :param env_file: path to the env file.
+    :return: None.
+    """
+    load_dotenv(dotenv_path=Path(env_file), override=False)
+
+
+def _verify_or_create_private_keys(ctx: Context) -> None:
+    """
+    Verify or create private keys.
+
+    :param ctx: Context
+    """
+    path = Path(DEFAULT_AEA_CONFIG_FILE)
+    agent_loader = ConfigLoader("aea-config_schema.json", AgentConfig)
+    fp = path.open(mode="r", encoding="utf-8")
+    aea_conf = agent_loader.load(fp)
+
+    for identifier, _value in aea_conf.private_key_paths.read_all():
+        if identifier not in SUPPORTED_CRYPTOS:
+            ValueError("Unsupported identifier in private key paths.")
+
+    fetchai_private_key_path = aea_conf.private_key_paths.read(FETCHAI)
+    if fetchai_private_key_path is None:
+        _create_fetchai_private_key()
+        aea_conf.private_key_paths.update(FETCHAI, FETCHAI_PRIVATE_KEY_FILE)
+    else:
         try:
-            ctx.protocol_loader.load(open(protocol_dir / DEFAULT_PROTOCOL_CONFIG_FILE))
-        except FileNotFoundError:
+            _try_validate_fet_private_key_path(fetchai_private_key_path)
```

Comment:
this could be simplified with `FetchAICrypto.load`

Reply:
https://github.com/fetchai/agents-aea/issues/881